### ISSUE TITLE
New peerDependencies issues

### ISSUE THREAD

**Issue by konsumer** [brianzinn/react-babylonjs#134]
I have seen this issue on a lot of node projects that have peer-deps with node > 15.

The quick-hack solution:
```
npm i --legacy-peer-deps
```

I was recently setting up a storybook with this project, so I clone it and run `npm i` and get this error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: react-babylonjs@3.0.15
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   dev react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.x.x" from @inlet/react-pixi@1.3.2
npm ERR! node_modules/@inlet/react-pixi
npm ERR!   dev @inlet/react-pixi@"^1.2.8" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/konsumer/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/konsumer/.npm/_logs/2021-05-21T21_54_07_380Z-debug.log
```

Here is output of `npm version` on Pop!OS (very similar to Ubuntu) 20.10:
```js
{
  'react-babylonjs': '3.0.15',
  npm: '6.14.13',
  ares: '1.17.1',
  brotli: '1.0.9',
  cldr: '39.0',
  icu: '69.1',
  llhttp: '6.0.1',
  modules: '93',
  napi: '8',
  nghttp2: '1.42.0',
  nghttp3: '0.1.0-DEV',
  ngtcp2: '0.1.0-DEV',
  node: '16.1.0',
  openssl: '1.1.1k+quic',
  tz: '2021a',
  unicode: '13.0',
  uv: '1.41.0',
  v8: '9.0.257.24-node.11',
  zlib: '1.2.11'
}
```

Generally, when I encounter this on other things, I have to go through and update deps, and even remove some peer-deps, or just recommend the `--legacy-peer-deps` flag.

**Comment by brianzinn** [comment]
this will be solved when I move storybook out of main project - the dev dependencies are just too much with storybook as part of main repo.  going to close, but thanks for sharing the issue.

**Comment by brianzinn** [comment]
just confirmed it is fixed now with node 16.6.2.
